Common/Product/SharedProject/ProjectNode.cs

```diff
@@ -1575,9 +1577,11 @@ internal enum EventTriggering {
                             }
 
                             FinishProjectCreation(basePath, baseLocation);
+                            this.LogProjectLoadTelemetry(TelemetryEvents.ProjectCreated);

```

Comment:
You shouldn't be adding anything to Common/*. This code is shared with other projects, like python tools. Instead, add to the classes that extend these ones (e.g. ```NodejsProjectNode```):
https://github.com/microsoft/visualstudio-sharedproject